Dockerfile

```diff
@@ -1,7 +1,17 @@
-FROM alpine:3.10
+FROM alpine:3.22
 RUN apk add --no-cache \
   bash \
   git \
-  ncurses
+  ncurses \
+  musl-dev \
+  wget \
+  findutils
+RUN wget -qO- https://api.github.com/repos/crate-ci/typos/releases/latest \
+  | grep "browser_download_url" \
+  | grep "x86_64-unknown-linux-musl" \
+  | cut -d '"' -f 4 \
+  | xargs wget
+RUN tar -xvf /typos-*.tar.gz
+ENV PATH="/:$PATH"
```

Comment:
Would it be a little simpler and less brittle to fetch it like this?
```
RUN wget -O- https://github.com/crate-ci/typos/releases/download/v1.34.0/typos-v1.34.0-x86_64-unknown-linux-musl.tar.gz \
  | tar -C/bin -xz ./typos
```

Reply:
Yeah, I just didn't know how important it was to always have the latest release. This brittle function will (until it breaks) make sure the latest tagged release is always the version we're running.

Reply:
:shrug:  probably ok?

Reply:
Getting the latest release sure would be nice to keep the typos list up to date.

Is `jq` available in the container in which the pr-validator is running? If so, it might be more appropriate than the chain of `grep` and `cut`?

Reply:
I tried to combine both suggestions and found it easier to do in two steps, hope that's okay.